addons/web/static/src/scss/popover.scss

```diff
@@ -0,0 +1,158 @@
+@keyframes slide-top {
```

Comment:
Just a naive interrogation and probably just of personal interest :D: aren't those animation doable with owl's t-transition directive ?

Reply:
it could work with them to add a class for the transition, though i don't think it's needed here:
- the transitions are direction specific, and therefore linked to the `--{direction}` classes anyway, which are added dynamically based on the direction prop and viewport visibility
- you'd still need to define your animations in css like done here